comp/core/workloadmeta/collectors/internal/remote/sbomcollector/sbom_collector.go

```diff
@@ -0,0 +1,180 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build trivy
+
+// Package sbomcollector implements the remote SBOM collector for
+// Workloadmeta.
+package sbomcollector
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/CycloneDX/cyclonedx-go"
+	"go.uber.org/fx"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/grpclog"
+
+	"github.com/DataDog/datadog-agent/comp/core/workloadmeta/collectors/internal/remote"
+	workloadmeta "github.com/DataDog/datadog-agent/comp/core/workloadmeta/def"
+	"github.com/DataDog/datadog-agent/pkg/config/model"
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+	sbompb "github.com/DataDog/datadog-agent/pkg/proto/pbgo/sbom"
+	"github.com/DataDog/datadog-agent/pkg/util/flavor"
+	grpcutil "github.com/DataDog/datadog-agent/pkg/util/grpc"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	collectorID       = "sbom-collector"
+	cacheValidityNoRT = 2 * time.Second
+)
+
+type client struct {
+	cl              sbompb.SBOMCollectorClient
+	parentCollector *streamHandler
```

Comment:
It seems we are not using this field. Do we need it?

Reply:
Indeed. Removed